.github/workflows/ci.yml

```diff
@@ -12,13 +12,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: purescript-contrib/setup-purescript@main
+      # - uses: purescript-contrib/setup-purescript@main
+      - uses: srghma/setup-purescript@main
```

Comment:
Are you waiting on something to be merged upstream?

Reply:
yes, waiting to support spago@next

https://github.com/purescript-contrib/setup-purescript/pull/39

Reply:
was merged